core/application/impl/configuration_storage_impl.cpp

```diff
@@ -55,7 +139,8 @@ namespace kagome::application {
                 ensure(genesis_tree.get_child_optional("raw")));
     boost::property_tree::ptree top_tree;
     // v0.7 format
```

Comment:
I think it is not v0.7 format anymore. It is rather v2.0 format

Reply:
`// v0.7+ format` 
Okay? =)